accepted/2023/owner-author-pmui.md

```diff
@@ -0,0 +1,214 @@
+# Display Trusted Owners on the PM UI Browse Tab
+<!-- Replace `Title` with an appropriate title for your design -->
+
+- Donnie Goodson ([donnie-msft](https://github.com/donnie-msft))
+- GitHub Issue <https://github.com/NuGet/Home/issues/12501>
+
+## Summary
+
+A Trusted Owner is defined as a package owner that's indicated by a package source's [Search API](https://learn.microsoft.com/nuget/api/search-query-service-resource#search-result), as opposed to the free-form field in the `.nuspec`.
+
+Introduce a UI affordance to emphasize which package owners shown in the Packages Manager UI (PM UI) are trusted by the package source.
+In the case of the `nuget.org` package source, each owner will render with a hyperlink typically to the owner's profile page (eg, `'https://www.nuget.org/profiles/<owner name>'`).
+
+Only show `author` when no Trusted Owner is available from the selected package sources in PM UI.
+
+## Motivation
+
+<!-- Why are we doing this? What pain points does this solve? What is the expected outcome? -->
+
+Customers need a clear understanding of who created a NuGet package in order to trust that package.
+The term "owner" has become overloaded as NuGet has evolved, which has caused confusion.
+The "author" metadata is also misleading as it's a free-form text field which doesn't imply any verified credentials.
+
+Customers will be able to identify Trusted Owners of packages on the Browse Tab in the package list search results and in the package details pane of PM UI.
+
+## Explanation
+
+### Functional explanation
+
+<!-- Explain the proposal as if it were already implemented and you're teaching it to another person. -->
+<!-- Introduce new concepts, functional designs with real life examples, and low-fidelity mockups or  pseudocode to show how this proposal would look. -->
+
+Instead of showing only `author` as the PM UI does today, the packages list will now show Owners when available from a package source, and only fallback to author when Owners are not available.
+For this iteration, a single package source must be selected (ie, not `All`) and the `Browse` tab must be selected.
+
+The Details Pane will show both Owner and Author for a selected package.
+
+The Owners in both of these panes will be shown as a hyperlink.
```

Comment:
What if owners is part of the the search response, but there's no OwnerDetailsTemplate?

Reply:
There are 2 possibilities, and the 2nd one is `PackageDetailsUriTemplate`.
https://github.com/NuGet/Home/blob/dev-donnie-msft-ownerAuthorSpec/accepted/2023/owner-author-pmui.md#packages-list-ui-for-ownerauthor